tlsfuzzer/timing_runner.py

```diff
@@ -78,6 +80,49 @@ def generate_log(self, run_only, run_exclude, repetitions):
 
         self.log.write()
 
+    @staticmethod
+    def _format_seconds(sec):  # pragma: no cover
+        elems = []
+        ms, sec = math.modf(sec)
+        sec = int(sec)
+        days, rem = divmod(sec, 60*60*24)
+        if days:
+            elems.append("{0}d".format(days))
+        hours, rem = divmod(rem, 60*60)
+        if hours:
```

Comment:
`1d 0h 0m 0s` looks better than `1d` and causes less "jumping" in length of string so `if hours or elems` is a better idea